### PR TITLE
Build/Publish OCI images + misc fixes

### PR DESCRIPTION
**Pull request by IgorTodorovskiIBM**
* Removes color escaping when redirecting to a file
* Adds initial support for creating OCI artifacts via the --oci option
* Also fixes zopen download where repos not ending with port would be considered

### CODE REVIEW

File: bin/zopen-build [hunk 1]
```diff
@@ -651,18 +659,88 @@ install()
...
+  fi
+  if [ -z $ZOPEN_IMAGE_REGISTRY_KEY_FILE ] || [ ! -r $ZOPEN_IMAGE_REGISTRY_KEY_FILE ]; then
+    printWarning "Environment variable ZOPEN_IMAGE_REGISTRY_KEY_FILE is needed to push an OCI image"
+    return 4;
+  fi
+  if [ -z "$(command -v zosadk)" ]; then
+    printWarning "zosadk is not present on your system"
+    return 4
+  fi
+  zosadk build -t "${name}:${version}" .
```
Comment (MikeFultonDev) [inline bin/zopen-build]: are people able to get access to 'zosadk' ?

File: bin/common.inc [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 defineColors() 
 {
-  if [ ! "${_BPX_TERMPATH-x}" = "OMVS" ] && [ -z "${NO_COLOR}" ] && [ ! "${FORCE_COLOR-x}" = "0" ]; then
+  if [ ! "${_BPX_TERMPATH-x}" = "OMVS" ] && [ -z "${NO_COLOR}" ] && [ ! "${FORCE_COLOR-x}" = "0" ] && [ -t 1 ] && [ -t 2 ]; then
```
Comment (MikeFultonDev) [inline bin/common.inc]: cool. learned something new!

File: bin/zopen-build (hunk 1 above)
Comment (MikeFultonDev) [reply bin/zopen-build]: @IgorTodorovskiIBM

File: bin/zopen-download [hunk 3]
```diff
@@ -80,7 +84,6 @@ for repo in $(echo ${repo_results}); do
     fi
     rm -f "${pax}"
     dirname=${pax%.pax.Z}
-    name=${repo%port}
```
Comment (MikeFultonDev) [inline bin/zopen-download]: where is name coming from now?
can you test this with a couple 'different type' of builds if you haven't already like xxhashport ?

Comment (IgorTodorovskiIBM) [reply bin/zopen-download]: I moved it up to line 57: https://github.com/ZOSOpenTools/utils/pull/38/files#diff-ae055bc1692ab016fe02d0e29a4c671e6c2d9edce53be8e72badb7774d3013baR57

File: bin/zopen-build (hunk 1 above)
Comment (IgorTodorovskiIBM) [reply bin/zopen-build]: I've generalized it so that by default it looks for podman, but it can be overriden

**Review by MikeFultonDev — APPROVED** [review]

**Pull request merged**